src/validators/LoginValidator.ts

```diff
@@ -0,0 +1,10 @@
+export const validateLogin = (email: String, password: String) => {
+
+    if(email.length == 0 || email.length > 64) {
+        throw new Error("Email is not valid");
+    }
```

Comment:
Is it just length we are checking or are going to go complex and add regex to check the format of the email?

Reply:
Nvm just tested it and someone can change the page and send that